bench/cli.py

```diff
@@ -240,3 +242,17 @@ def _chdir(*args, **kwargs):
 		return f(*args, **kwargs)
 
 	os.chdir = _chdir
+
+
+def setup_exception_handler():
+	from traceback import format_exception
+
+	def handle_exception(exc_type, exc_info, tb):
+		print("".join(generate_exc(exc_type, exc_info, tb)))
+
+	def generate_exc(exc_type, exc_info, tb):
+		for t in format_exception(exc_type, exc_info, tb):
+			if "/click/" not in t:
+				yield t
+
+	sys.excepthook = handle_exception
```

Comment:
I've been using Pop for over a year and haven't encountered the `apport_excepthook`. Nothing like you've shared pops up in my terminal lol

It doesn't spill over to Frappe/other app commands since the bench process (interpreter 1 - global) replaces itself with a new invocation (interpreter 2 - env) by the `EXECV` directive with "copied" `os.environ`.